packages/electron-updater/src/BaseUpdater.ts

```diff
@@ -24,6 +24,18 @@ export abstract class BaseUpdater extends AppUpdater {
     }
   }
 
+  quitAppAndInstall(isForceRunAfter = false): void {
+    const isInstalled = this.install(false, isForceRunAfter);
```

Comment:
Some of our customers have requirements that want to explicit install app and do not want to run the app immediately after installation. So can't use `quitAndInstall(true, false)`. 
Mike, What do you think about changing the name from `quitAppAndInstall` to` quitAppAndExplicitInstall`, it can be more clearly distinguished from the original method `quitAndInstall` and reduce the confusion when using it. I'm glad to add new commit.

Reply:
Honestly, I really don't like this logic in the original function that was written: `isSilent ? isForceRunAfter : true` as I believe it effectively can ignore `isForceRunAfter`. That doesn't feel right to me

Quick Q from a business logic perspective. I can understand not wanting to have the app run after installation, but why does your installation need to not be silent?

Reply:
Rrequirements come from customer(T＿T) .